tests/common.py

```diff
@@ -83,7 +85,8 @@ def get_conn(cls: type, shared: bool = True) -> Database:
             _database_instances[cls] = get_conn(cls, shared=False)
         return _database_instances[cls]
 
-    return connect(CONN_STRINGS[cls], N_THREADS)
+    con = sqeleton.connect.load_mixins(AbstractMixin_NormalizeValue)
```

Comment:
This was required to get the "NormalizeAsString" working, not sure if this is actually required?